source/_templates/cloud/python_installation.rst

```diff
@@ -0,0 +1,20 @@
+.. Copyright (C) 2015 Wazuh, Inc.
+
+The |service_module| module requires `Python 3 <https://www.python.org/downloads/>`__. Specifically, it's compatible with Python |py_cloud_cont_min|–|py_cloud_cont_max|. While later Python versions should work as well, we can't assure they are compatible.
```

Comment:
What about unifying `|service_module|` into `|service|`? They seem to going to share the same name.

Reply:
I did it like that at first but I wasn't sure if the cloud services names per se qualify to be in `code`. But if you think it could be the way, I will change it